docs/general/polkadot-vault.md

```diff
@@ -0,0 +1,126 @@
+|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                |
+|                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format) (not recommended to use the same account on different chains). For `n` chains, a minimum of `n` mnemonics are recommended. |
+|                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                    For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via QR code fountain.                                                                                    |
+|                                                                                          Currently, not all parachains are supported.                                                                                          |                                                                                                               Users can add all parachains if they have the wss endpoint and know [how to extract the chain specifications and metadata](#add-chains).                                                                                                                |
+| Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                         Metadata updates are always available to install as soon as they are released, but the installation process requires familiarity with Linux commands.                                                                                                         |
```

Comment:
Same here. As long as the metadata is provided by a third party (and the user has added the chain from them), they don't need to go through the manual process. Additionally to the above, one reason the user might want to do it themselves is that the provider doesn't update the metadata quickly (pretty much like Ledger).

So, I would rephrase it to sth like: 
"Metadata updates are always available to install as soon as they are released, either through the third-party provider, or manually, in which case the installation process requires familiarity with Linux commands."